addons/l10n_latam_check/models/l10n_latam_checkbook.py

```diff
@@ -0,0 +1,59 @@
+from odoo import fields, models, api, _
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class L10nLatamCheckbook(models.Model):
+
+    _name = 'l10n_latam.checkbook'
+    _description = 'Checkbook'
+
+    name = fields.Char(compute='_compute_name', store=True)
```

Comment:
`name_get`?
It doesn't look like you need to store the name

Reply:
I remember we discuss something about this.
As we need to search by this field we would need also a name_search. Maybe the implementation of the name_search wouln't be nice or easy because the name_get is not only using data of fields, is using also custom strings on it.

The stored make more sence now?

@william-andre

Reply:
@jco-odoo 
I've removed the computed stored field and added the name_get.
I really don't know whow to implement on a clrear why a name search by "deferred, Currents, eletronics" but I believe thas as you said it's not really needed. 

IMHO, as it is, is good enought on terms of usability and code complexity.